README.md

```diff
@@ -83,6 +85,60 @@ We throw in the constructor if we cannot contact the affinity server within `aff
 
 Similarly, the worker will fail jobs that it takes where it cannot communicate the outcome to an Affinity Server. This will cause an incident to be raised if the connection is not re-established.
 
+### Redis
+
+
+Zeebe Node Affinity also allow to use a Redis pub/sub system to distribute workflow outcomes to interested clients. The `zeebe-node-affinity` library provides `RedisAffinity` and the `createWorkflowInstanceWithAffinity` method that extends the `createWorkflow` method with a callback. This callback is executed in-memory with the final variable state of the workflow. It can be used like this:
+
+Create workflow with redis affinity:
+
+```typescript
+const { RedisAffinity } = require("zeebe-node-affinity") 
+
+const zbcRedis = new RedisAffinity(ZEEBE_GATEWAY, { host: REDIS_HOST, password: REDIS_AUTH });
+
+zbcRedis.createWorkflowInstanceWithAffinity({
+        bpmnProcessId: workflowName,
+        variables: {
+        correlationKey,
+        userInputText: userInput,
+        chatFinished,
+        },
+        cb: (message) => res.send(message),
```

Comment:
This example demonstrates replying to a REST req with the outcome of the workflow? 

If so, we should probably make that explicit and wrap it in a REST route handler.

Reply:
Yes your right! Modif done